app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -69,4 +71,11 @@ class BindingTariffClassificationConnector @Inject()(configuration: AppConfig, c
     client.GET[Seq[Event]](url)
   }
 
+  def search(search: Search, sort: Sort)(implicit hc: HeaderCarrier, binder: QueryStringBindable[String]): Future[Seq[Case]] = {
+    val searchString = if(search.isDefined) "&" + Search.bindable.unbind("", search) else ""
+    val sortString = Sort.bindable.unbind("sort_by", sort)
```

Comment:
how about the sort direction? (ascending vs descending)

Reply:
Hmm good point

Reply:
I know @pllaurado is implementing that in the back-end

Reply:
Done